src/main/java/uk/ac/ebi/eva/commons/models/data/Variant.java

```diff
@@ -135,6 +135,8 @@ public Variant(String chromosome, int start, int end, String reference, String a
         this.chromosome = chromosome;
         this.start = start;
         this.end = end;
+        reference = Objects.nonNull(reference) ? reference.toUpperCase() :  null;
```

Comment:
Guessing this will not mutate the arguments that are passed since String is a primitive type. Could you double check?

Reply:
String being an immutable class, it will not mutate the existing object but will create a new one when **"toUpperCase()"** is called